src/util/__tests__/dateIsBetween.test.ts

```diff
@@ -0,0 +1,11 @@
+import { dateIsBetween } from '../dateIsBetween';
+
+describe('dateIsBetween', () => {
+  it('should return true if date is between two dates', () => {
+    expect(dateIsBetween('2000-08-21', '2022-09-19')).toBe(true);
```

Comment:
Hmm, maybe we should enforce the locale settings?

Reply:
oh ya i pushed this too soon. I need to fix these tests. They were passing locally at one point. 🤦🏻‍♂️

Reply:
No worries :)